src/pages/Home.vue

```diff
@@ -148,6 +150,11 @@ export default {
         ? this.sessionData.purposeParams
         : this.purposeParams;
     },
+
+    /** Returns object containing network data about the user */
+    getUserNetworkData() {
```

Comment:
Same comment. Since only IP, method name should reflect that

Reply:
So the reason I named it this was, for now it's only IP address, but in the future we could want to capture extra data, so created an object to which we can add furthur fields.

User agent data is also network data no?

Reply:
No no. Just call it IP address since that's what we care about. The method name should reflect what we're trying to get out. If we need more things later, we can either add more methods, or add a `getNetworkData` method and submethods which will return individual values. Right now, the issue is that someone reading the method name won't know that what we care about is the IP. Makes sense?